Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -339,16 +340,16 @@ class TabTrayViewController: UIViewController, Themeable {
 
     private func updateToolbarItems(forSyncTabs showSyncItems: Bool = false) {
         if shouldUseiPadSetup() {
-            if navigationMenu.selectedSegmentIndex == TabTrayViewModel.Segment.syncedTabs.rawValue {
-                navigationItem.rightBarButtonItems = (showSyncItems ? [doneButton, fixedSpace, syncTabButton] : [doneButton])
+            if segmentedControlIphone.selectedSegmentIndex == TabTrayViewModel.Segment.syncedTabs.rawValue {
```

Comment:
This func is hard to read there are a few `if cases` and it looks like Sync Tab has custom configuration I think we could have a private func for sync that returns the `rightBarButtonItems` to help with readability and we could unit test